diff -uprN a/bootable/recovery/Android.mk b/bootable/recovery/Android.mk
--- a/bootable/recovery/Android.mk	2024-02-04 18:04:00.000000000 +0800
+++ b/bootable/recovery/Android.mk	2024-02-04 18:05:00.000000000 +0800
@@ -621,6 +621,11 @@ LOCAL_POST_INSTALL_CMD += \
     sed -i "s/{themeversion}/$(TW_THEME_VERSION)/" $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)splash.xml; \
     sed -i "s/{themeversion}/$(TW_THEME_VERSION)/" $(TARGET_RECOVERY_ROOT_OUT)$(TWRES_PATH)ui.xml;
 
+# Debug
+LOCAL_CFLAGS += -g
+LOCAL_CPPFLAGS += -g
+LOCAL_STRIP_MODULE := false
+
 include $(BUILD_EXECUTABLE)
 
 # Symlink for file_contexts
