diff -uprN a/device/ONDA/octopus-ibt/octopus_ibt.mk b/device/ONDA/octopus-ibt/octopus_ibt.mk
--- a/device/ONDA/octopus-ibt/octopus_ibt.mk
+++ b/device/ONDA/octopus-ibt/octopus_ibt.mk
@@ -140,7 +140,8 @@ PRODUCT_AAPT_CONFIG := large xlarge hdpi xhdpi
 PRODUCT_AAPT_PERF_CONFIG := xhdpi
 PRODUCT_CHARACTERISTICS := tablet
 
-$(call inherit-product-if-exists, vendor/google/products/gms_base.mk)
+# Prebuilt webview
+PRODUCT_PREBUILT_WEBVIEWCHROMIUM := yes
 
 PRODUCT_BRAND := ONDA
 PRODUCT_NAME := octopus_ibt
diff --uprN a/device/ONDA/octopus-ibt/overlay/frameworks/base/core/res/res/values/config.xml b/device/ONDA/octopus-ibt/overlay/frameworks/base/core/res/res/values/config.xml
--- a/device/ONDA/octopus-ibt/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/device/ONDA/octopus-ibt/overlay/frameworks/base/core/res/res/values/config.xml
@@ -242,4 +242,7 @@
     <!-- Switch for lockscreen rotation by -->
     <bool name="config_enableLockScreenRotation">true</bool>
 
+    <!-- Use prebuilt webview -->
+    <string name="config_webViewPackageName" translatable="false">com.google.android.webview</string>
+
 </resources>
diff -uprN a/external/chromium_org/Android.mk b/external/chromium_org/Android.mk
--- a/external/chromium_org/Android.mk
+++ b/external/chromium_org/Android.mk
@@ -15,6 +15,7 @@ GYP_CONFIGURATION := Release
 # If the product is using a prebuilt libwebviewchromium then we need to skip
 # including most of the makefiles to avoid duplicate target definitions. Some
 # targets are still needed, so list their makefiles explicitly.
+PRODUCT_PREBUILT_WEBVIEWCHROMIUM := yes
 ifeq ($(PRODUCT_PREBUILT_WEBVIEWCHROMIUM),yes)
 GYP_MAKEFILE_STEMS := \
        $(addprefix $(CHROMIUM_DIR)/v8/tools/gyp/, \
